### DISCUSSION TITLE
恐怖作品（遊戲、電影）帶來的證明影響

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1560] [General]
酷愛恐怖類型作品的人​
在面對疫情時，心理往往調適得更好​
而那些偏好末日浩劫或喪屍危機題材的人​
不只心理韌性比較強、抗壓性比較高​
連為疫情做的準備也會比別人完善​
（比如很早就開始儲備物資）​
​雖然對大多數人來說​
玩恐怖遊戲或看恐怖片只是為了娛樂​
但我覺得這類作品其實提供了充分的學習環境​
能讓我們以極低的成本來探索我們不能體驗的世界

**Comment by poweichen00** [comment]
現在非常多實況主也都在玩恐怖遊戲，觀眾都想看實況主的效果，但我覺得要設計出好的恐怖遊戲難度非常高，不但要有創意、有趣的故事情節，在市面上的競爭對手也很多。

**Comment by youtinglin** [comment]
我覺得會看這種恐怖類型的電影的人，或許對世界的改變有著極大的承受力，對於世界有很多的想像，因此世界的變化對他們而言是很能接受的，而且研究證實看恐怖電影會對大腦有所刺激，也能活化大腦，增強對這世界的想像。

**Comment by aa103967622** [comment]
我覺得至少以看過恐怖片，比如說末日喪屍主題的，這一些人在面對真實情況的時候可能會擁有較高的抗壓性，因為他們在觀影過程中，有過類似的模擬體驗，這樣能讓他們更快進入狀況！

**Comment by larry40305** [comment]
喜愛恐怖類型作品的人可能對於危機情境更加敏感，他們可能更容易預見可能發生的危機並作出相應的準備。這種準備可能包括儲備物資、制定應急計劃等，以應對可能的災難狀況。這種行為可能是由於他們對危機情境的理解和對風險的敏感性。
